lib/readyset/configuration.rb

```diff
@@ -3,11 +3,20 @@
 
 module Readyset
   class Configuration
-    attr_accessor :migration_path, :shard
+    attr_accessor :enable_failover, :failover_error_window_period, :failover_error_window_size,
+      :failover_healthcheck_interval, :migration_path, :shard
 
     def initialize
+      @enable_failover = false
+      @failover_healthcheck_interval = 5.seconds
+      @failover_error_window_period = 1.minute
+      @failover_error_window_size = 10
```

Comment:
**_suggestion:_** What do you think about a team-up for this collection of parameters? 

![image](https://github.com/readysettech/readyset-rails/assets/22648572/64e22643-86cc-44e6-bde8-0b298391e752)

--- 

It seems like all of these options depend on each other. An abstraction that "summarizes" them could help out! `automatic_fallover_config_options`

Reply:
Makes sense to me!